backend/danswer/db/connector.py

```diff
@@ -270,7 +268,7 @@ def create_initial_default_connector(db_session: Session) -> None:
         input_type=InputType.LOAD_STATE,
         connector_specific_config={},
         refresh_freq=None,
-        prune_freq=None,
+        prune_freq=DEFAULT_PRUNING_FREQ,
```

Comment:
Why is this default now? What happens for file connector etc?

Reply:
This will only apply for the default Ingestion API, so shouldn't cause any issues elsewhere